files/build_templates/docker_image_ctl.j2

```diff
@@ -327,6 +327,24 @@ function postStartAction()
             docker cp $PSENSOR pmon:/usr/bin/
         fi
     fi
+
+{%- elif docker_container_name == "eventd" %}
+    # Wait until eventd container state is Running
+    until [[ ($(docker inspect -f {{"'{{.State.Running}}'"}} eventd) == "true") ]]; do
```

Comment:
>until [](http://example.com/codeflow?start=4&length=5)

Worried about the infinity loop. Do you have other more straightforward flow? how about move the copy statement into inside eventd container, maybe in start script?

Reply:
New change: We now mount /etc/rsyslog.d/ to eventd container. Inside of start script in eventd container, rsyslog plugin files will be copied to mounted /etc/rsyslog.d/ directory.  After eventd docker container starts, rsyslog service is restarted with new configuration. Please advise on new change.